cpp/src/Ice/LocalException.cpp

```diff
@@ -1,9 +1,513 @@
 // Copyright (c) ZeroC, Inc.
 
+#if defined(_MSC_VER)
```

Comment:
This code was largely moved from Exception.cpp. In particular, there was no change to the stack trace capture logic.